src/xrGame/script_game_object_inventory_owner.cpp

```diff
@@ -1138,7 +1138,6 @@ void CScriptGameObject::RestoreWeapon()
 {
 #ifdef DEBUG
     GEnv.ScriptEngine->script_log(LuaMessageType::Message, "CScriptGameObject::RestoreWeapon called!!!");
-    GEnv.ScriptEngine->print_stack();
```

Comment:
This change is not relevant to this PR, please, revert the change. I'll remove this debug code in separate commit :)